packages/tests/server/validators.test.ts

```diff
@@ -221,6 +223,32 @@ test('myzod', async () => {
   await close();
 });
 
+test('effect schema', async () => {
+  const t = initTRPC.create();
+
+  const router = t.router({
+    num: t.procedure
+      .input(toSelfValidatingSchema(S.number))
```

Comment:
Without knowing more context regarding effect, why is this 3rd party import needed? Are effect schemas not parsable on their own?

Reply:
If the schema has to be transformed for every usage anyway, maybe it makes more sense to write a `toZodLike` which would make it compatible with tRPC by default and useful with other parts of the ecosystem?

adding support for a new arbitrary validator format doesn't seem to make much sense to me if it doesn't add out the box support

Reply:
Agree

Reply:
Got it, I've submit a feature request at
https://github.com/Effect-TS/schema/issues/260